src/main/java/cz/cvut/kbss/termit/persistence/dao/comment/CommentDao.java

```diff
@@ -0,0 +1,126 @@
+package cz.cvut.kbss.termit.persistence.dao.comment;
+
+import cz.cvut.kbss.jopa.model.EntityManager;
+import cz.cvut.kbss.jopa.model.descriptors.Descriptor;
+import cz.cvut.kbss.jopa.model.descriptors.EntityDescriptor;
+import cz.cvut.kbss.jopa.model.descriptors.FieldDescriptor;
+import cz.cvut.kbss.termit.exception.PersistenceException;
+import cz.cvut.kbss.termit.model.Asset;
+import cz.cvut.kbss.termit.model.comment.Comment;
+import cz.cvut.kbss.termit.persistence.DescriptorFactory;
+import cz.cvut.kbss.termit.util.ConfigParam;
+import cz.cvut.kbss.termit.util.Configuration;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import java.net.URI;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@Repository
+public class CommentDao {
+
+    private final Descriptor loadingDescriptor;
+    private final Descriptor savingDescriptor;
+
+    private final EntityManager em;
+
+    @Autowired
+    public CommentDao(EntityManager em, DescriptorFactory descriptorFactory, Configuration config) {
+        this.em = em;
+        this.loadingDescriptor = createLoadingDescriptor(config.get(ConfigParam.COMMENTS_CONTEXT), descriptorFactory);
+        this.savingDescriptor = createSavingDescriptor(config.get(ConfigParam.COMMENTS_CONTEXT), descriptorFactory);
+    }
+
+    private Descriptor createLoadingDescriptor(String context, DescriptorFactory descriptorFactory) {
+        final EntityDescriptor descriptor = new EntityDescriptor(URI.create(context), false);
+        descriptor.addAttributeContext(descriptorFactory.fieldSpec(Comment.class, "author"), null);
```

Comment:
author is also inferred?

Reply:
Author is not inferred, but is in the default context.